http/rest-client-reactive/src/main/resources/httpVersion.properties

```diff
@@ -0,0 +1,6 @@
+// server settings
+quarkus.http.http2=true
```

Comment:
According to the guide: https://es.quarkus.io/guides/http-reference#http2-support , the http2 property is already set by default, so you can remove it.